src/Button.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+//  task-11
+function RestaurantButton(props) {
+  return (
+    <button className="btn btn-primary" onClick={props.handleClick}>
+      Add
+    </button>
+  );
```

Comment:
this is good, you're passing your handleClick function to your child component to use, this means you can update the state in the parent component